CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Changelog
 
+## [1.62](https://github.com/brave/brave-ios/releases/tag/v1.62)
+
+### Web3
+
+- Added support for fetching unknown tokens from non-mainnet transactions. [(#8529)](https://github.com/brave/brave-ios/issues/8529)
+- Added support for SVG blockies v2. [(#8375)](https://github.com/brave/brave-ios/issues/8375)
+- Added “Creating Wallet” step to Wallet Onboarding. [(#8345)](https://github.com/brave/brave-ios/issues/8345)
+- Added “Accounts” and “Transactions” tab to the asset detail view. [(#8137)](https://github.com/brave/brave-ios/issues/8137)
+- Added line chart v2 support for Wallet portfolio and asset details.  [(#8136)](https://github.com/brave/brave-ios/issues/8136)
```

Comment:
Maybe reword it to `Updated Line chart color to match the new Brave Wallet design on Portfolio and Asset details screen.`